src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -0,0 +1,302 @@
+local campaignTokenData = {
```

Comment:
I think you can omit almost all of these entries and it should use the default values instead